exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestCTAS.java

```diff
@@ -357,6 +358,39 @@ public void testTableIsFoundWithinWorkspaceWhenNameStartsWithSlash() throws Exce
     }
   }
 
+  @Test
+  public void testCtasForList() throws Exception {
+    String tableName = "testCtasForList";
+    try {
+      test("CREATE TABLE `%s`.`%s` AS SELECT l FROM cp.`jsoninput/input2.json`", DFS_TMP_SCHEMA, tableName);
+      testBuilder()
+          .sqlQuery("SELECT * FROM `%s`.`%s` LIMIT 1", DFS_TMP_SCHEMA, "/" + tableName)
```

Comment:
Since we already implicitly using string builder, having concatenation for the second parameter looks weird.